src/CssToInlineStyles.php

```diff
@@ -100,15 +100,9 @@ public function getInlineStyles(\DOMElement $element)
      */
     protected function createDomDocumentFromHtml($html)
     {
-        $html = trim($html);
-        if (strstr('<?xml', $html) !== 0) {
-            $xmlHeader = '<?xml encoding="utf-8" ?>';
-            $html = $xmlHeader . $html;
-        }
-
         $document = new \DOMDocument('1.0', 'UTF-8');
         $internalErrors = libxml_use_internal_errors(true);
-        $document->loadHTML($html);
+        $document->loadHTML(mb_convert_encoding($html, 'HTML-ENTITIES', 'UTF-8'));
```

Comment:
@stof otherwise the utf-8 encoding doesn't work correctly for some reason.

[Example](http://sandbox.onlinephpfunctions.com/code/c3c39251bb666815ecff9eda6d14b2166c09ffd3)